draft-ietf-taps-interface.md

```diff
@@ -633,6 +633,14 @@ In case a unidirectional connection is requested, but unidirectional connections
 the system should fall back to bidirectional transport.
 
 
+### Timeout for aborting Connection Establishment {#conn-establish-timeout}
+
+Type:
+: Integer
+
+This property specifies how long to wait before aborting a Connection during establishment.
+It is given in seconds.
```

Comment:
Really seconds here? I can imagine time-critical applications that want to use timeouts on the range of 200-500ms

Reply:
Good one!  We specify it in seconds, yet don't allow floats  :-D    For now, I'll just remove the statement about seconds and the Integer type here, and from "Timeout for aborting Connection", which has the same mistake.